pinot-core/src/main/java/org/apache/pinot/core/util/CrcUtils.java

```diff
@@ -81,8 +86,9 @@ public long computeCrc()
         }
       }
     }
-
-    return checksum.getValue();
+    long crc = checksum.getValue();
+    LOGGER.info("Computed crc = {}", crc);
```

Comment:
You might want to include all the files used to generate the crc for debugging purpose? Only logging crc does not provide a lot of value IMO. Same for MD5